platform/nativescript/compiler/modules/class.js

```diff
@@ -4,17 +4,17 @@ import { parseText } from 'compiler/parser/text-parser'
 function transformNode(el, options) {
   const warn = options.warn || baseWarn
   const staticClass = getAndRemoveAttr(el, 'class')
-  if (process.env.NODE_ENV !== 'production' && staticClass) {
-    const expression = parseText(staticClass, options.delimiters)
-    if (expression) {
-      warn(
-        `class="${staticClass}": ` +
-          'Interpolation inside attributes has been removed. ' +
-          'Use v-bind or the colon shorthand instead. For example, ' +
-          'instead of <div class="{{ val }}">, use <div :class="val">.'
-      )
-    }
-  }
+  // if (process.env.NODE_ENV !== 'production' && staticClass) {
+  //   const expression = parseText(staticClass, options.delimiters)
+  //   if (expression) {
+  //     warn(
+  //       `class="${staticClass}": ` +
```

Comment:
@farfromrefug any reason this has been removed? Interpolation in attributes should not be used - and should be warned about

Reply:
Well it was a deprecation warning that was implying a regexp test on every transform node. Seemed like it was time to remove it. But i can bring it back if you prefer

Reply:
@farfromrefug Just checked and it is still present in vue (web), so let's keep it there until even vue removes it! :)

https://github.com/vuejs/vue/blob/ebc1893faccd1a9d953a8e8feddcb49cf1b9004d/src/platforms/web/compiler/modules/class.js#L16-L22

Reply:
fixed